controller/src/main/java/io/pravega/controller/store/stream/tables/CommittingTransactionsRecord.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.controller.store.stream.tables;
+
+import io.pravega.common.ObjectBuilder;
+import io.pravega.controller.store.stream.tables.serializers.CommittingTransactionsRecordSerializer;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.SneakyThrows;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.UUID;
+
+@Data
+@Builder
+@AllArgsConstructor
+public class CommittingTransactionsRecord {
```

Comment:
Please add a detailed description of what this new record is supposed to mean. I asked you for a clarification offline, and I think a similar explanation would be nice as a preamble to this class.